drawbicyc/drawbicyc.cpp

```diff
@@ -19,7 +19,7 @@ int main(int argc, char** argv)
     ("qv_qc_2_6_10_min", po::value<bool>()->default_value(false) , "plot comparison of qv and qc fields at 2, 6 and 10 min?")
     ("dir", po::value<std::string>()->required() , "directory containing out_lgrngn")
     ("micro", po::value<std::string>()->required(), "one of: blk_1m, blk_2m, lgrngn")
-    ("type", po::value<std::string>()->required(), "one of: dycoms, moist_thermal, rico, Lasher_Trapp")//, base_prflux_vs_clhght")
+    ("type", po::value<std::string>()->required(), "one of: dycoms, moist_thermal, rico, Lasher_Trapp, ICMW2020")//, base_prflux_vs_clhght")
```

Comment:
change ICMW2020 to ICMW2020_cc to distinguish that this is the cumulus congestus case